### PR TITLE
PIM-5664 : Add documentation for purge jobs executions command

### PR DESCRIPTION
**Pull request by momoss**

### CODE REVIEW

File: cookbook/job_execution.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+The batch job command
+---------------------
+
+Akeneo PIM provides a simple command to purge jobs executions:
+
+.. code-block:: bash
+    :linenos:
+
+    app/console akeneo:batch:purge-job-execution [-d|--days DAYS]
```
Comment (damien-carcel) [inline cookbook/job_execution.rst]: As you explained the option “--days” just below, I am not sure it is relevant to put it here

Comment (momoss) [inline cookbook/job_execution.rst]: Just follow others batch command examples, but I can remove it

File: cookbook/job_execution.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    :linenos:
+
+    crontab -e
+
+You can now add a new line at the end of the opened file:
+
+.. code-block:: bash
+    :linenos:
+
+    0 * * * * /home/akeneo/pim/app/console akeneo:batch:purge-job-execution --env=prod
```
Comment (nidup) [inline cookbook/job_execution.rst]: once a day is enough for this kind of cleanup task

File: cookbook/index.rst [hunk 3]
```diff
@@ -28,3 +28,4 @@ Cookbook
     localization/index
     rule/index
     setup_data/add_translation_packs
+    job_execution
```
Comment (nidup) [inline cookbook/index.rst]: purge_job_executions

File: cookbook/job_execution.rst [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+With this cron configuration a purge of jobs executions older than 90 days, will be launched every hour.
```
Comment (nidup) [inline cookbook/job_execution.rst]: once a day would be better

**Comment by nidup** [comment]
small comments then GTM

**Pull request merged**